modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-cr-by-using-cli_{context}"]
+= Configuring the NotificationsConfiguration CR by using CLI
+
+You can configure the `NotificationsConfiguration` CR by using CLI.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance.
```

Comment:
I think this prereq should be worded consistently with the same prereq in the other module:
```suggestion
* You have enabled notifications for the Argo CD instance. For more information, see "Enabling notifications with an Argo CD instance".
```

(My suggested text does not have the xref link since that's not allowed in a module)

Reply:
Fixed.